src/app/views/search-route/search-route.component.css

```diff
@@ -68,23 +68,23 @@
   top: 0;
   left: 0;
   z-index: 100;
-  background-color: rgba(0,0,0,.2);
+  background-color: rgba(0, 0, 0, 0.2);
 }
 
 .search-route__route-information {
   display: flex;
   align-items: center;
   background-color: var(--secondary);
   color: white;
-  box-shadow: 0 2px 12px 0 rgba(0,0,0,.2);
+  box-shadow: 0 2px 12px 0 rgba(0, 0, 0, 0.2);
   padding: 1rem;
   width: 100%;
   z-index: 100;
 }
 
 .search-route__locations {
   flex-grow: 1;
-  margin: 0 .5rem;
+  margin: 0.5rem;
```

Comment:
did you mean to change this to `margin: 0.5rem;` or should it be `margin: 0 0.5rem;`